package.json

```diff
@@ -1,31 +1,29 @@
 {
   "name": "react-final-form-html5-validation",
   "version": "1.0.0",
-  "description":
-    "A swap-in replacement for 🏁 React Final Form's <Field> component to provide HTML5 Validation",
+  "description": "A swap-in replacement for 🏁 React Final Form's <Field> component to provide HTML5 Validation",
```

Comment:
I think my npm formats package.json when doing a fresh install for some reason, no idea how to stop it (havent searched for it though 😉 ). Hope it's ok